src/docs/asciidoc/getting-started-eventuate-tram-sagas.adoc

```diff
@@ -62,20 +62,48 @@ repositories {
 
 If you are writing a Saga orchestrator add this dependency to your project:
 
-* `io.eventuate.tram.sagas:eventuate-tram-sagas-simple-dsl:$eventuateTramSagasVersion`
+===== Spring
+
+* `io.eventuate.tram.sagas:eventuate-tram-sagas-spring-orchestration-simple-dsl:$eventuateTramSagasVersion`
+
+===== Micronaut
+
+* `io.eventuate.tram.sagas:eventuate-tram-sagas-orchestration-simple-dsl:$eventuateTramSagasVersion`
+* `io.eventuate.tram.sagas:eventuate-tram-sagas-micronaut-orchestration:$eventuateTramSagasVersion`
 
 If you are writing a saga participant then add this dependency:
 
-* `io.eventuate.tram.sagas:eventuate-jpa-sagas-framework:$eventuateTramSagasVersion`
+===== Spring
+
+* `io.eventuate.tram.sagas:eventuate-tram-sagas-spring-participant:$eventuateTramSagasVersion`
+
+===== Micronaut
+
+* `io.eventuate.tram.sagas:eventuate-tram-sagas-micronaut-participant:$eventuateTramSagasVersion`
+
 
 You must also include one of the https://github.com/eventuate-tram/eventuate-tram-core[Eventuate Tram] 'implementation' artifacts:
 
-* `io.eventuate.tram.core:eventuate-tram-jdbc-kafka:$eventuateTramVersion` - JDBC database and Apache Kafka message broker
-* `io.eventuate.tram.core:eventuate-tram-in-memory:$eventuateTramVersion` - In-memory JDBC database and in-memory messaging for testing
+===== Spring
+
+* `io.eventuate.tram.core:eventuate-tram-spring-jdbc-kafka:$eventuateTramVersion` - JDBC database and Apache Kafka message broker
```

Comment:
There are others: Redis, ActiveMQ,.. right?

Reply:
HM...It seems so, but as I remember, only activemq is tested besides kafka.

Reply:
Testing is a separate issue. Tram Sagas uses Tram so there no reason why it won't work with any message broker supported by Tram.

Reply:
ok, I will add them